config/settings-production.yml

```diff
@@ -15,6 +15,20 @@ buffy:
       hidden: true
     thanks:
       hidden: true
+    welcome:
+      if:
+        value_matches:
+          submission-type: "Standard|Estándar|Stats"
+      message: "Thanks for submitting to rOpenSci, our editors and @ropensci-review-bot will reply soon. Type `@ropensci-review-bot help` for help."
```

Comment:
the message looks good, the only suggestion I have would be to have a link to the page with the editorial board.